hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java

```diff
@@ -89,29 +73,7 @@
 @Tag(CRYPTO)
 @HapiTestLifecycle
 @DisplayName("Token airdrop")
-public class TokenAirdropTest {
-    private static final String OWNER = "owner";
-    // receivers
-    private static final String RECEIVER_WITH_UNLIMITED_AUTO_ASSOCIATIONS = "receiverWithUnlimitedAutoAssociations";
-    private static final String RECEIVER_WITH_FREE_AUTO_ASSOCIATIONS = "receiverWithFreeAutoAssociations";
-    private static final String RECEIVER_WITHOUT_FREE_AUTO_ASSOCIATIONS = "receiverWithoutFreeAutoAssociations";
-    private static final String RECEIVER_WITH_0_AUTO_ASSOCIATIONS = "receiverWith0AutoAssociations";
-    private static final String ASSOCIATED_RECEIVER = "associatedReceiver";
-    // tokens
-    private static final String FUNGIBLE_TOKEN = "fungibleToken";
-    private static final String NON_FUNGIBLE_TOKEN = "nonFungibleToken";
-    // tokens with custom fees
-    private static final String FT_WITH_HBAR_FIXED_FEE = "fungibleTokenWithHbarCustomFee";
-    private static final String FT_WITH_HTS_FIXED_FEE = "fungibleTokenWithHtsCustomFee";
-    private static final String FT_WITH_FRACTIONAL_FEE = "fungibleTokenWithFractionalFee";
-    private static final String NFT_WITH_HTS_FIXED_FEE = "NftWithHtsFixedFee";
-    private static final String NFT_WITH_ROYALTY_FEE = "NftWithRoyaltyFee";
-    private static final String DENOM_TOKEN = "denomToken";
-    private static final String HTS_COLLECTOR = "htsCollector";
-    private static final String HTS_COLLECTOR2 = "htsCollector2";
-    private static final String HBAR_COLLECTOR = "hbarCollector";
-    private static final String TREASURY_FOR_CUSTOM_FEE_TOKENS = "treasuryForCustomFeeTokens";
-    private static final String OWNER_OF_TOKENS_WITH_CUSTOM_FEES = "ownerOfTokensWithCustomFees";
+public class TokenAirdropTest extends TokenAirdropBase {
```

Comment:
I highly recommend adding atleast happyPathTest in the same PR before merging

Reply:
I currently implement these tests https://github.com/hashgraph/hedera-services/issues/14672 but I'm stuck with one of them. Hopefully, I will be done today and push the tests.

Reply:
The tests are pushed 👍